lib/dock_health_api/resources/usergroup.rb

```diff
@@ -0,0 +1,15 @@
+module DockHealthApi
+    class UserGroup < Resource
+      extend DockHealthApi::Crud::Create
+      extend DockHealthApi::Crud::Delete
+      extend DockHealthApi::Crud::Get
+      extend DockHealthApi::Crud::Update
+      extend DockHealthApi::Crud::List
+  
+      def self.resource_url
+        "#{client.config.resource_url}/api/#{url_version}/user/group"
+      end
+
+    end
+  end
```

Comment:
what is this indentation 😂

Reply:
@kgoodwin-mdl , do we need all of these endpoints for usergroups?

Reply:
less endpoints means less specs 😎

Reply:
Ill fix that, it was copy paste error from the other file.

Not sure if we need all of them but they are implemented by DH